### ISSUE TITLE
Move aria component descriptions to under the component headers

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2084]
### Subject of the issue
As navigating by heading will skip the component descriptions, the should be moved accordingly. See #2083 

### Actions
* Move component description into component.hbs under header
* Remove component description declarations from components
* Add flag to turn on and off component description per component instance

**Comment by oliverfoster** [comment]
Complete
